src/contrast/pages/UploadNewPaper/ExperimentsSection/Findings.jsx

```diff
@@ -634,12 +633,24 @@ export default function Findings({
           </div>
         );
       })}
-      <AddFieldButton
-        disabled={disableAddBttns}
-        initialValues={initialValues}
-        fieldValues={fieldValues}
-        setFieldValues={setFieldValues}
-      />
+      <div className="w-full flex justify-center">
+        <Tooltip
+          placement="top"
+          content={
+            !disableAddBttns
+              ? "Add findings field"
+              : !editble.every((value) => value === false)
```

Comment:
I hate nested ternary. I suggest moving to a function that would resolve the logic, texts, etc